cmake/Config.cmake.in

```diff
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/LLGLTargets.cmake")
+
+check_required_components(@PROJECT_NAME@)
```

Comment:
I'm not familiar with this type of CMake configuration. Is this the one necessary to support `find_package`? Could you please share a small example how this would be used in another CMake project? I previously added a separate `.cmake` file but haven't used it in a while, such as [cmake/FindLLGL.cmake](https://github.com/LukasBanana/LLGL/blob/master/cmake/FindLLGL.cmake). You would include this file to your `CMakeLists.txt` script and then use the CMake variables such as `LLGL_INCLUDE_DIR` etc.

Reply:
Yes. It is from here: https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html#exporting-targets

Usage would be
```cmake
find_package(LLGL CONFIG REQUIRED)
target_link_libraries(main PRIVATE LLGL::LLGL_OpenGL)
```

Reply:
Cool, I tried your fork and I can see the exported `LLGLTargets.cmake` file. But how would this be picked up by another project? This file is located in a hashed subfolder (in my case `CMakeFiles\Export\0d73479e6c054791440a2c54bad87ff6`) of the installation directory. Would a client have to copy that manually or how do you direct CMake to find that file? Or is that something the [vcpkg](https://github.com/Microsoft/vcpkg/) package manager takes care of? I never used that tool.

Reply:
If I do 
```
cmake -S . -B build -GNinja  
cmake --build build    
cmake --install build --prefix dest
```
Then it is not installed to a hashed sub folder. It gets installed to `lib/cmake/LLGL/`. If you want to use the package you set the `CMAKE_PREFIX_PATH` to the folder to which you installed llgl (here it would be dest)